backend/app/assets/stylesheets/spree/backend/shared/_tables.scss

```diff
@@ -4,27 +4,9 @@ table {
   border-spacing: 0;
 
   th, td {
-    border-right: 1px solid $color-border;
-    border-bottom: 1px solid $color-border;
-    vertical-align: middle;
+    vertical-align: top;
```

Comment:
A bunch of these (this particular example included) can be removed and replaced with the inline editing solution that Martin worked on. I linked a drive document in #2042 for those. I'm ok with this style for the ones that remain.